Scroller/tests/ScrollerContentContainerDecorator-specs.js

```diff
@@ -0,0 +1,26 @@
+import '@testing-library/jest-dom';
```

Comment:
Let's rename this file as we changed the name of the decorator. :)

Reply:
Oh, I changed the file name to the ContentContainerDecorator-specs.js.